src/expandableCalendar/WeekCalendar/presenter.ts

```diff
@@ -62,6 +62,9 @@ class Presenter {
   };
 
   shouldComponentUpdate = (context: any, prevContext: any) => {
+    if (!context) {
```

Comment:
I would love to get rid of this function and on a more general note - I would love to get rid of this presenter (or at least to have it for pure functions alone).